SUMMARY.md

```diff
@@ -5,6 +5,7 @@
 * Usage
   * [Getting Started](docs/Usage/GettingStarted.md)
   * [Examples](docs/Usage/Examples.md)
+  * [Cells](docs/Usage/Cells.md)
```

Comment:
If you guys find the order of Cells.md is strange, just let me know :)

Reply:
IMO Cells should come before Examples

Reply:
Sure, addressed.